src/node/history.h

```diff
@@ -329,6 +334,8 @@ namespace ccf
 
     void emit_signature() override
     {
+#ifndef PBFT
+      // Signatures are only emitted when Raft is used as consensus
```

Comment:
Do we not want signatures in the ledger with PBFT?

Reply:
That's a very good point. AIUI, signatures are useful in Raft to assign blame but also to provide general ledger integrity. The trust assumptions for PBFT are certainly different but we still need a mechanism to provide ledger integrity. I've disabled signatures with PBFT for now because the way they are generated will probably be different (i.e. go through a round of PBFT before being committed). @achamayou and @ashamis will be able to provide more details on this.

Reply:
At the moment, the PBFT ledger is going to look like https://github.com/microsoft/CCF/tree/ledger, and it won't include signatures just yet. When the signatures are included, they'll be somewhat different anyway. It's likely that the general form of a ledger entry will be something like:

request entry = request, [ { node signature } ]
write set = write set, [ { node signature } ]
response = response, { node signature }
ledger entry = [ request entry ], write set, [ response ]

Reply:
So far we've not been too specific about signatures, we have "client signatures" and the ledger/raft signatures are just called "signatures". Perhaps it's time to call them "ledger signatures". Ideas for better names?

Reply:
Thanks for the clarification. As discussed, they could be called ledger signatures, raft signatures or leader signatures.